src/widget/OktaSignIn.js

```diff
@@ -153,7 +153,7 @@ var OktaSignIn = (function () {
     var authClient = createAuthClient(authParams);
 
     var Router;
-    if (options.stateToken && !Util.isV1StateToken(options.stateToken)) {
+    if ((options.stateToken && !Util.isV1StateToken(options.stateToken)) || options.proxyIdxResponse) {
```

Comment:
I would change this to 
```
(options.stateToken || options.proxyIdxResponse) && !Util.isV1StateToken(options.stateToken
```

Reply:
I'd prefer what Yannong did. :)

Reply:
In that case if it has v1stateToken & contains `proxyIdxResponse` then it will still go to V2 pipeline..  right ?

Reply:
I think so.

Reply:
I feel if there is v1StateToken then it should only go to V1 router ? and just ignore `proxyIdxResponse` param..

Reply:
I think either way will work, just how we want to handle v1/v2 state token use cases when proxyIdxResponse is present.

Reply:
since there is no concept of `proxyIdxResponse` for V1.. I would assume it's always for v2 pipeline only.. @haishengwu-okta what do you think ?

Reply:
already update this part based on @pradeepdewda-okta 's advice